Lib/test/lock_tests.py

```diff
@@ -370,6 +370,19 @@ def test_locked(self):
         lock.release()
         self.assertFalse(lock.locked())
 
+    def test_locked_2threads(self):
+        l = []
+        rlock = self.locktype()
+        def acquire():
+            rlock.acquire()
```

Comment:
Can you add `l.append(rlock.locked())` before and check `l[0]` (false) and `l[1]` (true) below?